### DISCUSSION TITLE
Ordinals on Dogecoin chain?

### DISCUSSION THREAD

**Discussion by suiAKT** [dogecoin/dogecoin#3225] [General]
Ordinals on Dogecoin chain? Lets discuss

**Comment by patricklodder** [comment]
What for?

**Reply by LouisLibre** [reply]
Doge is a meme.
Memes should be in Doge as well.
Meme economy is good for Doge.

**Reply by patricklodder** [reply]
Doge is a meme, Doge**coin** is a coin. Sounds like you're talking about Doge**file**, which is the meme sharing network favored by Shiba Inus world wide, when you're done building. Recommended reading for your success: http://www.drivechain.info

If you're willing to embrace separation of concerns, I'm willing to help realize it in a way that is much wow.

**Reply by suiAKT** [reply]
ordinlas on dogecoin chains like NFT on chain. Recently on btc chain it was a success, as dogecoin is a fork it should be possible here too.

**Reply by Siva0702** [reply]
can i have link to inscribe 1 dogecoin ordinal

**Reply by jgadipriss** [reply]
it looks like the joke got out of hand and became one of the top in the industry.

**Reply by suiAKT** [reply]
https://shibescription.xyz/

Here you can inscribeordinals on dogecoin. "doginals"
